src/mudlet.cpp

```diff
@@ -2584,6 +2584,17 @@ void mudlet::deleteLine(Host* pHost, const QString& name)
     }
 }
 
+std::optional<QSize> mudlet::getImageSize(const QString& imageLocation)
+{
+    QImage image(imageLocation);
```

Comment:
Apropos a previous comment - `imageLocation` is supposed to be a pathFileName so would it not be reasonable to check that it *is* a valid, readable file first, BEFORE trying to read it as an image?

Reply:
Qt will try and fail, and that's what the null check is for later.

Reply:
Ah, but won't the image be null both if the file does not exist OR it is an empty (null) image?

Reply:
We'll still say we didn't find an image in that very rare case and people will debug from there on